docs/toml-schema.md

```diff
@@ -202,6 +202,20 @@ excluded-people = [
     "rylev",
 ]
 
+# Define Zulip streams owned
+# It's optional, and there can be more than one
+[[zulip-streams]]
+# The name of the Zulip stream (required)
+name = "t-overlords"
+# Zulip groups that will have access to the stream if it is private (optional)
```

Comment:
I'm not sure what we want the behavior of this to be when the stream isn't private. Also, given that this stream is defined on the team itself, would any of the `zulip-groups` automatically be included here?